docker/src/main/java/clocker/docker/entity/container/application/VanillaDockerApplicationSshDriver.java

```diff
@@ -63,6 +65,16 @@ public void install() {
     }
 
     @Override
+    public void copyRuntimeResources() {
+        Map<String, String> runtimeFiles = entity.getConfig(VanillaSoftwareProcess.RUNTIME_FILES);
+        Map<String, String> runtimeTemplates = entity.getConfig(VanillaSoftwareProcess.RUNTIME_TEMPLATES);
+        if ((runtimeFiles != null && runtimeFiles.size() > 0) ||
+                (runtimeTemplates != null && runtimeTemplates.size() > 0)) {
+            super.copyRuntimeResources();
```

Comment:
Should we change Brooklyn's `AbstractSoftwareProcessDriver`, so that it only does the `createDirectory(getRunDir(), "create run directory")` if there are runtime resources to copy? (I presume nothing downstream in normal Brooklyn entities relies on runDir existing because of `copyRuntimeResources`, but we'd need to check that).